frontend/src/pages/NewPipelineVersion.tsx

```diff
@@ -109,7 +109,7 @@ const descriptionCustomRenderer: React.FC<CustomRendererProps<string>> = props =
   return <Description description={props.value || ''} forceInline={true} />;
 };
 
-class NewPipelineVersion extends Page<{}, NewPipelineVersionState> {
+class NewPipelineVersion extends Page<{ namespace?: string }, NewPipelineVersionState> {
```

Comment:
where is this namespace coming from?
In other pages that use the namespace, it is read from the context, see example: https://github.com/kubeflow/pipelines/blob/ad419cd63dc9a6c7bc109c509311bdec5f671e57/frontend/src/pages/NewRun.tsx#L1202-L1204

Reply:
Yes, you're correct - will fix

Reply:
I believe this is resolved with commit https://github.com/kubeflow/pipelines/pull/6291/commits/bd9935cb0fe6c3210a093719fd4b7201566f73e9

I copied the format from the referenced `NewRun` by creating the `EnhancedNewPipelineVersion` and changing the default to `EnhancedNewPipelineVersion`